lib/android_safetynet/attestation_response.rb

```diff
@@ -46,7 +46,7 @@ def certificate_chain
     private
 
     def valid_nonce?(nonce)
-      payload["nonce"] == nonce
+      WebAuthn::SecurityUtils.secure_compare(payload["nonce"], nonce)
```

Comment:
Note to ourselves: The separate gem won't have this method, unless it's copied from WebAuthn or moved to `securecompare` gem.